tssc/step_implementers/push_artifacts/maven.py

```diff
@@ -1,5 +1,48 @@
-"""
-Step Implementer for the push-artifacts step for Maven.
+"""Step Implementer for the push-artifacts step for Maven.
+
+Step Configuration
+------------------
+
+Step configuration expected as input to this step.
+Could come from either configuration file or
+from runtime configuration.
+
+| Configuration Key | Required? | Default | Description
+|-------------------|-----------|---------|-----------
+| `url`             | True      |         | URL to the artifact repository to push the artifact to.
+| `user`            | False     |         | User to authenticate with the artifact repository.
+| `password`        | False     |         | Password to authenticate with the artifact repository.
+
+
+Expected Previous Step Results
+------------------------------
+
+Results expected from previous steps that this step requires.
+
+| Step Name           | Result Key  | Description
+|---------------------|-------------|------------
+| `generate-metadata` | `version`   | TODO
+| `package`           | `artifacts` |
+
+Results
+-------
+
+Results output by this step.
+
+| Result Key  | Description
+|-------------|------------
+| `artifacts` | An array of dictionaries with information on the pushed artifacts.
+
+**artifacts**
+Keys in the dictionary elements in the `artifacts` array in the step results.
+
+| `artifacts`Key | Description
```

Comment:
@itewk Minor space issue -> 'artifacts'_Key

Reply:
thanks. fixing in new commit.